src/js/pages/Home.js

```diff
@@ -1,49 +1,74 @@
 import React from "react";
+var $ = require('jquery');
+var _ = require('lodash');
 
-import Home from "../components/Home";
-import * as HomeActions from "../actions/HomeActions";
-import HomeStore from "../stores/HomeStore";
+import BroadcastMessage from "../components/BroadcastMessage";
 
+import Events from "../components/Events"
+import EventStore from "../stores/EventStore";
 
 export default class Featured extends React.Component {
-  constructor() {
+  constructor(props) {
     super();
-    this.getHome = this.getHome.bind(this);
     this.state = {
-      home: HomeStore.getAll(),
+      messages: [],
+      events: [],
+      apiBaseURL: props.route.apiBaseURL,
     };
   }
 
-  componentWillMount() {
-    HomeStore.on("change", this.getHome);
-  }
+  componentDidMount() {
+    this.broadcastRequest = $.get(this.state.apiBaseURL + "/broadcast", function (result) {
+      this.setState({
+        messages: result.data,
+      });
+    }.bind(this));
 
-  componentWillUnmount() {
-    HomeStore.removeListener("change", this.getHome);
-  }
+    var event_ids = [];
+    var evnts = [];
+    var self = this;
+    this.eventRequest = $.get(this.state.apiBaseURL + "/user/1/subscription", function (result) {
```

Comment:
Use @larajanecka's EventStore? In my and Lara's code all the calls happen in a Store